LatinumNetwork/mosaic
# [BRAN-151] Added Nunito as font

## Purpose/Description:

- Added Nunito as font family by default.
- Added dependecy from npm for Nunito font.

## Screenshots:

![image](https://github.com/LatinumNetwork/mosaic/assets/1088165/7154e764-42aa-40fc-b4c2-bf314755cc7f)
## Jira Link:

[BRAN-151 Add Nunito Sans Font - Mosaic](https://collagegroup.atlassian.net/browse/BRAN-151)

## Dependencies:

No dependencies

## Test Plan:

-   **Test Scenario 1:** Run the project

    -   **Steps:**
        1. Select Typography
        2. Pick, default, header or body
        3. Use developer tools, pick inspector tab.
        4. Select text for default: Default, for header: Heading and body: Body
    -   **Expected Result:** In the css you should see `font-family: 'Nunito',sans-serif;` (as you can see in the attachment)
    
## Additional Notes

### Reviewer Instructions:

None

### Deployment Considerations:

None
